src/server/supplements/har/harTracer.ts

```diff
@@ -141,6 +141,7 @@ export class HarTracer {
     const pageEntry = this._ensurePageEntry(page);
     const harEntry: har.Entry = {
       pageref: pageEntry.id,
+      _id: request.id(),
```

Comment:
If har had this property, i would be ok with it. But since it is an extension, given that there is no good use case for this story, i'm reluctant to letting it in.

Reply:
@pavelfeldman I empathize with the hesitation. If it's helpful in your team's decision, here is what I was thinking while proposing this MR and why I felt it was appropriate:

1. Concrete, actionable use case. This isn't theoretical. With this field, we can delete buggy heuristic code in our codebase that we use to debug and do other slicing of the HAR data. Without this change, we either have to live with bugs or set out to re-implement all of the HarTracer code in user-land. (Another user/company (@vigneshshanmugam) sounds like they have a use case from Slack, too).
2. There's no way today, aside from heuristics, to tie a Request object you see in Node back with an entry in the HAR. (If there is, please let me know as that is the sole purpose of this PR. No need to add new API if already doable.)
3. HAR format is under spec'd. Every browser, HAR proxy, networking debug tool I've used adds fields that are useful to the user, so as long as they don't break people's code, I don't see the harm in adding them if they can be used within reason.
4. `playwright@next` (`playwright@1.15.0-next-alpha-sep-10-2021`) introduces 2 extension fields (that I don't see in Chromium [`_frameref`](https://source.chromium.org/search?q=_frameref&sq=&ss=chromium) and [`_monotonicTime`](https://source.chromium.org/search?q=_monotonicTime&ss=chromium)):
    ```
    "entries": [
      {
        "pageref": "page@702b0129b3d3993b3eb0fe38cae484ba",
        "_frameref": "frame@dfaf6341b6e76342a310147e06861f2b",     <-------------
        "_monotonicTime": 364665600.663, <---------------
    ```
    While these are useful for playwright internally, we'd find the `_id` extension field useful for us today. (Let me know if there's another place I should look for a standard.)

Would it make more sense to put it in the "connection" field?

> connection [string, optional] (new in 1.2) - Unique ID of the parent TCP/IP connection, can be the client or server port number. Note that a port number doesn't have to be unique identifier in cases where the port is shared for more connections. If the port isn't available for the application, any other unique connection ID can be used instead (e.g. connection index). Leave out this field if the application doesn't support this info.

source: http://www.softwareishard.com/blog/har-12-spec/